### ISSUE TITLE
Steals/Loses Focus

### ISSUE THREAD

**Issue by markcrobbins** [aviral-batra/obsidian-power-search#2]
I've tried changing the debouncer sliders to get rid of this but when I type or highlight and the results pane is open, eventually the focus goes somewhere and I can't get it back even with 'focus on editor' command or tabbing around to discover where I might be - even 'close pane' does not respond. No hotkey will respond, and the only way to get back focus is to click into a note body or some other focusable UI.

**Comment by aviral-batra** [comment]
Is it only a problem when this plugin is open? I have not been able to reproduce this issue.

**Comment by markcrobbins** [comment]
Yes, only when the results pane is open, so I thought the focus was going somewhere there, but tabbing around does not focus any elements ever as far as I can tell.
